[Fix][Doc] Fix LocalFile doc (#7887)

YOMO-Lee opened the pull request:
Supplement and optimize the description of the LocalFile connector on filtering files [(#7887)](https://github.com/apache/seatunnel/issues/7887)

Supplement and optimize the explanation and description of the LocalFile connector regarding file filtering

YOMO-Lee committed:
[Fix][Doc] Fix LocalFile doc (#7887)

Supplement and optimize the description of the LocalFile connector on filtering files
[(#7887)](https://github.com/apache/seatunnel/issues/7887)

YOMO-Lee committed:
Merge branch 'apache:dev' into dev

Hisoka-X commented on docs/en/connector-v2/source/LocalFile.md:
```diff
@@ -254,6 +254,12 @@ Specifies Whether to process data using the tag attribute format.
 
 Filter pattern, which used for filtering files.
 
+The filtering format is similar to wildcard matching file names in Linux.
+
+However, it should be noted that unlike Linux wildcard characters, when encountering file suffixes, the middle dot cannot be omitted.
+
+For example, `abc20241022.csv`, the normal Linux wildcard `abc*` is sufficient, but here we need to use `abc*.*` , Pay attention to a point in the middle.
```
```suggestion
Matching rules use regular expression rules.
```
I think we should tell user what rules we follow.

YOMO-Lee replied:
我觉得只说正则，用户容易产生疑问，比如我，用了正则一直不生效，最好有个例子

Hisoka-X replied:
Yes, we can add some demo, but we should tell user what rules we follow at first. Please go ahead and add some demo. For example:
```
File Structure Example:
/project/documents/report.txt
/project/documents/notes.txt
/project/data/input.csv
/project/data/processed/output.csv
/project/data/archive/old_data.csv
/project/images/logo.png
/project/scripts/script.sh
/project/scripts/utils/helpers.sh

Matching Rules Example:
Example 1: Match all .txt files
Regular Expression:
.*/.*\.txt$

Example 2:


```

YOMO-Lee replied:
好的，我一会完善下

Hisoka-X reviewed (commented):
Thanks @YOMO-Lee for raise this PR. Please modify all File series doc.

YOMO-Lee reviewed (commented):
例子在下面也有说明

YOMO-Lee committed:
Merge branch 'apache:dev' into dev

YOMO-Lee committed:
[Fix][Connector-V2][ClickHouse] Fix ClickHouse Bug (#7897)

1、When the ClickHouse connector is set to multi parallelism, the task extraction is completed but cannot be stopped normally
[(#7897)](https://github.com/apache/seatunnel/issues/7897)

2、Added E2E test cases for this issue [(#7897)](https://github.com/apache/seatunnel/issues/7897)

3、Local developers want to observe **Job Progress Information** in a timely manner,  Need to modify the following configuration.The configuration in config is invalid
```
seatunnel engine/seatunnel-engineer-common/src/main/resources/seatunnely.yaml
```

YOMO-Lee committed:
[Fix][Connector-V2][ClickHouse] Fix ClickHouse Bug (#7897)

1、When the ClickHouse connector is set to multi parallelism, the task extraction is completed but cannot be stopped normally
[(#7897)](https://github.com/apache/seatunnel/issues/7897)

2、Added E2E test cases for this issue [(#7897)](https://github.com/apache/seatunnel/issues/7897)

3、Local developers want to observe **Job Progress Information** in a timely manner,  Need to modify the following configuration.The configuration in config is invalid
```
seatunnel engine/seatunnel-engineer-common/src/main/resources/seatunnely.yaml
```

YOMO-Lee committed:
[Fix][Connector-V2][ClickHouse] Fix ClickHouse Bug (#7897)

1、When the ClickHouse connector is set to multi parallelism, the task extraction is completed but cannot be stopped normally
[(#7897)](https://github.com/apache/seatunnel/issues/7897)

2、Added E2E test cases for this issue [(#7897)](https://github.com/apache/seatunnel/issues/7897)

3、Local developers want to observe **Job Progress Information** in a timely manner, Need to modify the following configuration.The configuration in config is invalid
```
seatunnel engine/seatunnel-engineer-common/src/main/resources/seatunnely.yaml
```
